rpm/tzdata-timed.spec

```diff
@@ -1,12 +1,11 @@
 Name:       tzdata-timed
 
 Summary:    Data files for the time daemon, timed
-Version:    2017b
+Version:    2021a
 Release:    1
-Group:      System/System Control
 License:    Public Domain
```

Comment:
Should this have a license file anyway?

Reply:
Hm, technically there is no license if it's public domain? Some COPYING type of file dunno, but that's separate anyway from the update.

Reply:
It's just that to me not having a license of any kind sounds problematic.